SIMID/examples/maps/player/simid_player.html

```diff
@@ -0,0 +1,117 @@
+ <html>
```

Comment:
Add some text to the maps/README file. Can be short, just explain what the creative will be and what files will go in that dir

Reply:
done!